doc/source/python/python_server.md

```diff
@@ -198,3 +198,4 @@ Python Server can be configured using environmental variables or command line fl
 | N/A                     | `FILTER_METRICS_ACCESS_LOGS`           | `not debug`   | Filter out logs related to Prometheus accessing the metrics port. By default enabled in production and disabled in debug mode.                                     |
 | N/A                     | `PREDICTIVE_UNIT_METRICS_ENDPOINT`     | `/metrics`    | Endpoint name for Prometheus metrics. In k8s deployment default is `/prometheus`.                                                                                  |
 | N/A                     | `PAYLOAD_PASSTHROUGH`                  | `false`       | Skip decoding of payloads.                                                                                                                                         |
+| `--accesslog`           | `SELDON_GUNICORN_ACCESS_LOG`           | `false`       | Enable gunicorn access log.                                                                                                               |
```

Comment:
CLI flag could be `--access-log` - I think this would fit better style of rest of flags.

Reply:
Yeah will update